repo_e/capital_letter.py

```diff
@@ -0,0 +1,7 @@
+def capital(strings):
+    Capital_str=[]
+    for i, element in enumerate(strings):
+        if  element==element.capitalize():
```

Comment:
please make a habit of formatting your code with black

Reply:
Ok